src/app/backend/resource/event/common.go

```diff
@@ -136,13 +137,20 @@ func GetNodeEvents(client kubernetes.Interface, dsQuery *dataselect.DataSelectQu
 		return &eventList, err
 	}
 
-	events, err := client.CoreV1().Events(v1.NamespaceAll).Search(scheme, node)
+	eventsInvUID, err := client.CoreV1().Events(v1.NamespaceAll).Search(scheme, node)
 	_, criticalError := errors.HandleError(err)
 	if criticalError != nil {
 		return &eventList, criticalError
 	}
 
-	eventList = CreateEventList(FillEventsType(events.Items), dsQuery)
+	node.UID = types.UID(node.Name)
+	eventsInvName, err := client.CoreV1().Events(v1.NamespaceAll).Search(scheme, node)
+	_, criticalError = errors.HandleError(err)
+	if criticalError != nil {
+		return &eventList, criticalError
+	}
+
+	eventList = CreateEventList(FillEventsType(append(eventsInvName.Items, eventsInvUID.Items...)), dsQuery)
```

Comment:
Isn't there a chance of getting duplicate entries due to 2 event calls? Shouldn't we check for duplicates and filter them out from the result array?

Reply:
> Isn't there a chance of getting duplicate entries due to 2 event calls? Shouldn't we check for duplicates and filter them out from the result array?

I don't think it is necessary. One event will not be set to `node.uid` and `node.name` to be saved separately. The `node.name` is the name we entered, and then the uid is randomly generated. Uid consists of 32-bit letters or numbers. It is almost impossible that the two are equal.

Reply:
Makes sense